client/src/product-components/AddToCart/AddToCart.jsx

```diff
@@ -47,39 +57,53 @@ class AddToCart extends React.Component {
     });
   }
 
+  handleAddToCart() {
+    axios.post('/cart', {
+      sku_id: this.props.sizeList[this.state.selectedIndex].sku,
+    })
+    .then(function (response) {
```

Comment:
When it succeeds console log, when it fails console log? 

If you have to do this then at least a .finally